templates/_common.tpl

```diff
@@ -64,15 +76,30 @@ volumes:
     secret:
       secretName: {{ include "movedValue" (dict "Values" .Values "Key" "coderd.tls.devurlsHostSecretName") }}
 {{- end }}
+{{- if ne .Values.postgres.ssl.certSecret.name "" }}
+  - name: pgcert
+    secret:
+      secretName: {{ .Values.postgres.ssl.certSecret.name | quote }}
+{{- end }}
+{{- if ne .Values.postgres.ssl.keySecret.name "" }}
+  - name: pgkey
+    secret:
+      secretName: {{ .Values.postgres.ssl.keySecret.name | quote }}
+{{- end }}
+{{- if ne .Values.postgres.ssl.rootCertSecret.name "" }}
+  - name: pgrootcert
+    secret:
+      secretName: {{ .Values.postgres.ssl.rootCertSecret.name | quote }}
+{{- end }}
 {{- end }}
 
 {{/* 
   coder.volumeMounts adds a volume mounts stanza if a cert.secret is provided.
 */}}
 {{- define "coder.volumeMounts" }}
-{{- if or (merge .Values dict | dig "certs" "secret" "name" false) (ne (include "movedValue" (dict "Values" .Values "Key" "coderd.tls.hostSecretName")) "") }}
 volumeMounts:
-{{- end }}
+  - name: tmp-pgcerts
+    mountPath: /tmp/pgcerts
```

Comment:
maybe just mount this as /tmp so that other things can use it?

Reply:
We can't mount at `/tmp` because that's where the coderd binary is. There's no reason that binary can't be moved in the future, but that's for another PR.

Reply:
ah ok